assets/src/blocks/amp-story-post-author/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getMetaBlockSettings } from '../../helpers';
+import './edit.css';
+
+export const name = 'amp/amp-story-post-author';
+
+export const settings = {
+	title: __( 'Story Author', 'amp' ),
+	description: __( 'Displays the name of the story\'s author', 'amp' ),
+	category: 'common',
+	icon: 'admin-users',
+	keywords: [
+		__( 'post', 'amp' ),
+		__( 'author', 'amp' ),
+	],
+	...getMetaBlockSettings( {
```

Comment:
Since this block could also be considered to be a “Byline” block, there may be a desire to show "By" before the name. Perhaps a toggle to include “By” before the name? This would prevent users from having to try to carefully position a separate Text block that just contains “By”. But in that case, the user can just use a Text block for the while byline manually if they want in the immediate term.

Reply:
Good idea.

Since I already created [a card for further customizations of these blocks](https://github.com/ampproject/amp-wp/projects/10#card-20136438), I added this suggestion to the list. It requires some thinking of how to best make the meta blocks extensible in an easy way without duplicating too much code. Which is why I think we should tackle it in a separate PR.

Reply:
Good. Let's address later. Not critical.